src/controllers/router-store/index.tsx

```diff
@@ -398,6 +443,33 @@ const actions: AllRouterActions = {
         history[updateType](updatedRelativePath);
       }
     },
+  loadRoute:
```

Comment:
This action name is a bit confusing as might be interpreted as loading the full route (while instead we just load the loader 😅 . So `loadLoader` is not great either, but maybe `runLoaderLoad` ? 🤔

Reply:
I suggested the `loadRoute` name because in the case of EntryPoints (or any custom loader) you load both the component and data. It also does not have to be limited to this, it could load other things within the route (e.g. other components related to the page)